.travis.yml

```diff
@@ -10,17 +10,20 @@ python:
 cache:
   pip: true
   directories:
-    - node_modules
+    - coalahtml/_coalahtml/node_modules
+    - coalahtml/_coalahtml/bower_components
 
 before_install:
```

Comment:
Earlier Travis detected that it shold do `npm install` by default, now it wont ...